certbot/client.py

```diff
@@ -498,6 +498,10 @@ def deploy_certificate(self, domains, privkey_path,
                            "the certificate")
             raise errors.Error("No installer available")
 
+        logger.warning("Future versions of Certbot will automatically configure "
+            "the webserver so that all requests redirect to secure HTTPS access. "
+            "If you do not want this feature, run Certbot with the --no-redirect flag.")
```

Comment:
The way this is worded almost seems like it's discouraging people to set up redirects which I don't think we should do. I'm pretty open on the exact wording but what about something like:

> You can control this behavior and disable this warning with the --redirect and --no-redirect flags.

(That's assuming we implement my above suggestion.)

Alternatively, we could actively push people towards setting up redirects.

Reply:
I was actually trying to push them *towards* setting up redirects here, by saying "if you don't want this feature". Your wording works for me though.

Reply:
Oh. I didn't interpret it that way because it's telling people how to disable the feature, but not how to get it now. I personally prefer my wording over what you initially wrote in this PR, but if there's something else you like better, please propose it!

Reply:
Nope, I'm cool with whatever.